src/sentry/features/__init__.py

```diff
@@ -179,6 +179,7 @@
 default_manager.add("organizations:session-replay-sdk-errors-only", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:session-replay-recording-scrubbing", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:session-replay-index-subquery", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:session-replay-weekly-email", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
can't use our remote handler for this as its too high volume. can use an `INTERNAL` strategy and we can specify one in getsentry